src/graphql/query/promotionalBanner.graphql

```diff
@@ -13,22 +13,34 @@ query {
 		}
 		lendingRewardOffered
 	}
-	holiday_enabled: setting(setting: "feature.holiday_mode.enabled") {
-		value
+	holiday_enabled: general {
+		setting(setting: "feature.holiday_mode.enabled") {
+			value
+		}
 	}
-	holiday_start_time: setting(setting: "feature.holiday_mode.start_time") {
-		value
+	holiday_start_time: general {
+		setting(setting: "feature.holiday_mode.start_time") {
+			value
+		}
 	}
-	holiday_end_time: setting(setting: "feature.holiday_mode.end_time") {
-		value
+	holiday_end_time: general {
+		setting(setting: "feature.holiday_mode.end_time") {
+			value
+		}
 	}
-	promo_enabled: setting(setting: "promo.topnav_promo.enabled") {
-		value
+	promo_enabled: general {
+		setting(setting: "promo.topnav_promo.enabled") {
+			value
+		}
 	}
-	promo_start_time: setting(setting: "promo.topnav_promo.start_time") {
-		value
+	promo_start_time: general {
+		setting(setting: "promo.topnav_promo.start_time") {
+			value
+		}
 	}
-	promo_end_time: setting(setting: "promo.topnav_promo.end_time") {
-		value
+	promo_end_time: general {
+		setting(setting: "promo.topnav_promo.end_time") {
+			value
+		}
```

Comment:
Instead of aliasing the entire 'general' tree repeatedly, the aliases could be nested under 'general'
```
general {
  holiday_enabled: setting(setting: "feature.holiday_mode.enabled") {
    value
  }
  ...
}
```
and then the get in the js becomes `_get(data, 'general.holiday_enabled.value')`

Reply:
cool, can do. also found some test stuff I'm fixin  up now